florian_berlin/index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My hometown Berlin</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins|Sintony" rel="stylesheet">
+    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+</head>
+
+<body>
+
+    <nav>
+        <ul>
+            <li><a href="#history">history</a></li>
+            <li class="location"><a href="#location">location</a></li>
+            <li><a href="#images">images</a></li>
+            <li><a href="#sights">sights</a></li>
+        </ul>
+    </nav>
+
+    <div id="wrapper">
+
+        <header>
+            <h1>Berlin</h1>
+        </header>
+
+        <h2 id="history">history</h2>
+
+        <section class="history-section">
+
+            <div class="history-box1">
```

Comment:
Avoid using class names with numbers in them. A number is too spesific no? Either make it an id, cause ids are unique just like number =) or make it a generic class history-box that will fit all your history boxes. And if you want to customise on just make it by id